.github/workflows/testing.yaml

```
@@ -102,3 +103,11 @@ jobs:
       - run: make check
 
       - run: make test
+
+      - name: Send code coverage to 'coverage.io'
+        run: |
+          make test-coverage
+          make coveralls
```

The target `coveralls` depends on `test-coverage`. As a result at now tests with coverage runs twice.
```suggestion
      - name: Send code coverage to 'coverage.io'
        run: |
          make coveralls
```

---

Yes, now tests with coverage run once.